core/src/main/java/com/linecorp/armeria/client/ConnectionPoolListener.java

```diff
@@ -95,6 +99,10 @@ static ConnectionPoolListener metricCollecting(MeterRegistry registry) {
      *   <td>{@code <name>#count{state="closed"}}</td>
      *   <td>The number of closed connections.</td>
      * </tr>
+     * <tr>
+     *   <td>{@code #value{state="active"}}</td>
```

Comment:
To align with the above examples:
```suggestion
     *   <td>{@code <name>#value{state="active"}}</td>
```